src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js

```diff
@@ -18,3 +16,4 @@ export default class App extends Component {
     );
   }
 }
+App.displayName = App.name;
```

Comment:
Is this equivalent to leaving line 9 there but changing it to `static displayName = App.name`?

I don't really know if either way of expressing this is preferable over the other. Perhaps it makes no difference. I'm certainly happy for you to decide what's best.

Reply:
I like the static thing better, will send an update.